cmd/unused/internal/ui/interactive/delete_view.go

```diff
@@ -107,6 +112,11 @@ func (m deleteViewModel) View() string {
 	} else {
 		confirm := bold.Render("Press `x` to start deleting the following disks:")
 		fmt.Fprintf(sb, "You're about to delete %d disks from %s %s\n%s\n", len(m.disks), m.provider.Name(), m.provider.Meta(), confirm)
```

Comment:
based on screenshot it looks a bit confusing have both notes on the same screen.
wdyt about moving that part into `else` section for `if m.dryRun`?

Reply:
🤔 interesting point. I did it this way so the whole workflow works the same as it would if it wasn't in dry mode, but what you're suggesting made me realize something: say that we are in dry run mode and select lots of disks to delete; in this case it will only list things, and even if you press `x` to _delete_ them, it will do nothing, and then you would need to restart the app again, this time without passing `-n` to enable dry run, so you will need to re-select the disks. Would it make more sense that in this particular view we enable a key binding to toggle dry run mode off?

Reply:
yeah, something like "press D to Switch to DANGER mode".
as another option: in dry-run mode we could save selection into file and add flag to read "selection files list" from file. but that sounds like completely different and requiring more changes to the current workflow...

Reply:
Added a `D` key bind to toggle dry-run mode.